draft-ietf-ppm-dap.md

```diff
@@ -1683,42 +1624,8 @@ following checks:
    the same. If so, the Aggregator MUST mark the input share as invalid with
    error `invalid_message`.
 
-1. Check if the report may still be aggregated with the current aggregation
-   parameter. This can be done by looking up all aggregation parameters
-   previously used for this report and calling
-
-   ~~~
-   Vdaf.is_valid(current_agg_param, previous_agg_params)
```

Comment:
Is there a need for `Vdaf.is_valid` anymore? It _might_ be used to validate that an aggregation parameter itself is valid (irrespective of other aggregation parameters), but I think this might be better done at some point in evaluating the VDAF rather than via a separate method, for simplicity's sake.

Reply:
Yes, I agree. The check for whether a report has been previously aggregated at all should suffice, and VDAF methods like `prep_init` or `prep_next` should safely reject invalid aggregation parameters. It's still useful for `Vdaf.is_valid` to be defined, because DAP-HH would use it, but DAP doesn't need to call it anymore.

Reply:
I think of validity of the agg param as being delegated to decoding on the wire.